src/__tests__/assets.test.ts

```diff
@@ -115,6 +118,41 @@ describe('assets/', () => {
           }
         });
 
+        it('should contain the migration contracts networks in all other files', async () => {
```

Comment:
IMO this can be part of the above test, most of the counting is duplicated anyway.

Reply:
I initially thought so, but as the intention of the tests were different, thought to have it separated (one to make sure all the deployments are present in all contracts except the migration one, while the other test makes sure the migration test deployments are also present in other contracts.)